tests/performance_tests/performance_test_basic.py

```diff
@@ -552,6 +553,19 @@ def createBaseArgumentParser():
         ptbBaseParserGroup.add_argument("--contract-dir", type=str, help="Path to contract dir", default="unittests/contracts/eosio.system")
         ptbBaseParserGroup.add_argument("--wasm-file", type=str, help="WASM file name for contract", default="eosio.system.wasm")
         ptbBaseParserGroup.add_argument("--abi-file", type=str, help="ABI file name for contract", default="eosio.system.abi")
+        ptbBaseParserGroup.add_argument("--wasm-runtime", type=str, help="Override default WASM runtime (\"eos-vm-jit\", \"eos-vm\")\
+                                         \"eos-vm-jit\" : A WebAssembly runtime that compiles WebAssembly code to native x86 code prior to\
+                                         execution. \"eos-vm\" : A WebAssembly interpreter.",
+                                         choices=["eos-vm-jit", "eos-vm", "eos-vm-jit"], default="eos-vm-jit")
+        ptbBaseParserGroup.add_argument("--contracts-console", type=bool, help="print contract's output to console", default=False)
+        ptbBaseParserGroup.add_argument("--eos-vm-oc-cache-size-mb", type=int, help="Maximum size (in MiB) of the EOS VM OC code cache", default=1024)
+        ptbBaseParserGroup.add_argument("--eos-vm-oc-compile-threads", type=int, help="Number of threads to use for EOS VM OC tier-up", default=1)
+        ptbBaseParserGroup.add_argument("--eos-vm-oc-enable", type=bool, help="Enable EOS VM OC tier-up runtime", default=False)
```

Comment:
`eos-vm-oc-enable` would be for validation nodes but not production nodes. We need to handle this on a per node basis something like how we've used `specificExtraNodeosArgs` in the past.